packages/ifixit-api-client/client.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { useAppContext } from '@ifixit/app';
```

Comment:
I feel like we're better off transitioning this to use the config directly... rather than use a context. We don't have to do it here, but it seems silly to have and use a context when the string is available from the config.